src/plugins/data/common/field_formats/field_formats_registry.ts

```diff
@@ -19,42 +19,36 @@
 
 // eslint-disable-next-line max-classes-per-file
 import { forOwn, isFunction, memoize } from 'lodash';
-import { IUiSettingsClient, CoreSetup } from 'kibana/public';
+
+import { ES_FIELD_TYPES, KBN_FIELD_TYPES } from '../../common';
+
 import {
-  ES_FIELD_TYPES,
-  KBN_FIELD_TYPES,
+  GetConfigFn,
+  IFieldFormatConfig,
   FIELD_FORMAT_IDS,
   IFieldFormatType,
   IFieldFormatId,
-  FieldFormat,
   IFieldFormatMetaParams,
-} from '../../common';
-import { FieldType } from './types';
-
-export class FieldFormatRegisty {
-  private fieldFormats: Map<IFieldFormatId, IFieldFormatType>;
-  private uiSettings!: IUiSettingsClient;
-  private defaultMap: Record<string, FieldType>;
-  private basePath?: string;
-
-  constructor() {
-    this.fieldFormats = new Map();
-    this.defaultMap = {};
-  }
-
-  getConfig = (key: string, override?: any) => this.uiSettings.get(key, override);
-
-  init({ uiSettings, http }: CoreSetup) {
-    this.uiSettings = uiSettings;
-    this.basePath = http.basePath.get();
-
-    this.parseDefaultTypeMap(this.uiSettings.get('format:defaultTypeMap'));
-
-    this.uiSettings.getUpdate$().subscribe(({ key, newValue }) => {
-      if (key === 'format:defaultTypeMap') {
-        this.parseDefaultTypeMap(newValue);
-      }
-    });
+} from './types';
+import { baseFormatters } from './constants/base_formatters';
+import { FieldFormat } from './field_format';
+
+export class FieldFormatRegistry {
```

Comment:
nit: Everywhere we refer to this as `FieldFormats` -- field formats service, field formats registry (in the filename), but here is the one place we seem to use singular `FieldFormatRegistry` instead of plural `FieldFormatsRegistry`.

If we are already updating the references to it everywhere to correct the misspelling of `Registy`, perhaps worth it to make the names plural at the same time so everything is consistent?